api/scripts/createTable.ts

```diff
@@ -0,0 +1,58 @@
+import AWS from 'aws-sdk';
```

Comment:
Overall looks good to me. But is there any way we can check if table exists or not. If not exists then create. But may be this is not required for MVP but I will suggest you to put a @todo for future dev

Reply:
awesome. I'll update it